diff --git a/node_modules/node-pty/binding.gyp b/node_modules/node-pty/binding.gyp
index 79a93e7..30d853b 100644
--- a/node_modules/node-pty/binding.gyp
+++ b/node_modules/node-pty/binding.gyp
@@ -24,28 +24,28 @@
   'conditions': [
     ['OS=="win"', {
       'targets': [
-        {
-          'target_name': 'conpty',
-          'include_dirs' : [
-            '<!(node -e "require(\'nan\')")'
-          ],
-          'sources' : [
-            'src/win/conpty.cc',
-            'src/win/path_util.cc'
-          ],
-          'libraries': [
-            'shlwapi.lib'
-          ],
-        },
-        {
-          'target_name': 'conpty_console_list',
-          'include_dirs' : [
-            '<!(node -e "require(\'nan\')")'
-          ],
-          'sources' : [
-            'src/win/conpty_console_list.cc'
-          ],
-        },
+        # {
+        #   'target_name': 'conpty',
+        #   'include_dirs' : [
+        #     '<!(node -e "require(\'nan\')")'
+        #   ],
+        #   'sources' : [
+        #     'src/win/conpty.cc',
+        #     'src/win/path_util.cc'
+        #   ],
+        #   'libraries': [
+        #     'shlwapi.lib'
+        #   ],
+        # },
+        # {
+        #   'target_name': 'conpty_console_list',
+        #   'include_dirs' : [
+        #     '<!(node -e "require(\'nan\')")'
+        #   ],
+        #   'sources' : [
+        #     'src/win/conpty_console_list.cc'
+        #   ],
+        # },
         {
           'target_name': 'pty',
           'include_dirs' : [
diff --git a/node_modules/node-pty/deps/winpty/src/winpty.gyp b/node_modules/node-pty/deps/winpty/src/winpty.gyp
index 1e36e62..c67cb97 100644
--- a/node_modules/node-pty/deps/winpty/src/winpty.gyp
+++ b/node_modules/node-pty/deps/winpty/src/winpty.gyp
@@ -10,7 +10,7 @@
       'type': 'shared_library',
       'include_dirs': [
         'include',
-        '../misc/buffer-tests',
+        # '../misc/buffer-tests',
       ],
       'defines': [
         'COMPILING_WINPTY_DLL',
diff --git a/node_modules/node-pty/src/win/winpty.cc b/node_modules/node-pty/src/win/winpty.cc
index 2f4a06f..1e3e597 100644
--- a/node_modules/node-pty/src/win/winpty.cc
+++ b/node_modules/node-pty/src/win/winpty.cc
@@ -133,7 +133,7 @@ void init(Local<Object> target, Local<Value> ignored, Local<Context> context, v
   Nan::SetMethod(target, "startProcess", PtyStartProcess);
   Nan::SetMethod(target, "resize", PtyResize);
   Nan::SetMethod(target, "kill", PtyKill);
-  Nan::SetMethod(target, "getProcessList", PtyGetProcessList);
+  // Nan::SetMethod(target, "getProcessList", PtyGetProcessList);
   Nan::SetMethod(target, "getExitCode", PtyGetExitCode);
 };

